NuGet-Packages.md

```diff
@@ -2,25 +2,21 @@ NuGet is a standard package manager for .NET applications which is built into Vi
 
 Once you do a search, you should see a list similar to the one below (versions may be different, but names should be the same).
```

Comment:
We should add a link to https://aka.ms/wct-winui3 for info about WinUI 3 and the naming transition. Just add a blurb with a high-level summary.

We probably also want to call out the preview feeds briefly and just point to that Wiki Page as well.